### PR TITLE
Bump AAQ to v1.1.4, and rename the component name

### PR DESCRIPTION
**Pull request by nunnatsa**
## What this PR does / why we need it
AAQ now called "application-aware-quota". One of the configuration field name also renamed.

This PR changes the import path of AAQ to the new path, bumps it to v1.1.4, and rename the component and the
`enableClusterAppsResourceQuota` field to
`allowApplicationAwareClusterResourceQuota`, to match the rename in AAQ.

**Reviewer Checklist**

> Reviewers are supposed to review the PR for every aspect below one by one. To check an item means the PR is either "OK" or "Not Applicable" in terms of that item. All items are supposed to be checked before merging a PR. 

- [ ] PR Message
- [ ] Commit Messages
- [ ] How to test
- [ ] Unit Tests
- [ ] Functional Tests
- [ ] User Documentation
- [ ] Developer Documentation
- [ ] Upgrade Scenario
- [ ] Uninstallation Scenario
- [ ] Backward Compatibility
- [ ] Troubleshooting Friendly

**Jira Ticket**:

```jira-ticket
None
```

**Release note**:

```release-note
Bump AAQ to v1.1.4
```

### CODE REVIEW

File: docs/cluster-configuration.md [hunk 1]
```diff
@@ -1117,7 +1117,7 @@ With the `Custom` profile, the cipher list should be expressed according to Open
 
 On plain k8s, where APIServer CR is not available, the default value will be `Intermediate`.
 
-## Configure Applications Aware Quota (AAQ)
+## Configure Application Aware Quota (AAQ)
 To enable the AAQ feature, add the `applicationAwareConfig` field to the HyperConverged spec, with an empty object 
 or with custom settings (see below).
```
Comment (Barakmor1) [inline docs/cluster-configuration.md:1123]: Can you please change all the "ApplicationsResourceQuota" to "ApplicationAwareResourceQuota" (i see one in this file  line 1132)

Comment (nunnatsa) [reply docs/cluster-configuration.md:1123]: done

**Comment by vladikr** [comment]
Looks good to me

**Comment by nunnatsa** [comment]
hco-e2e-upgrade-prev-operator-sdk-aws lane passed
/override ci/prow/hco-e2e-upgrade-prev-operator-sdk-azure

**Review by orenc1 — APPROVED** [review]
/approve

**Pull request merged**